winpr/libwinpr/thread/test/TestThreadCreateProcess.c

```diff
@@ -15,7 +15,14 @@ int TestThreadCreateProcess(int argc, char* argv[])
 	BOOL status;
 	DWORD exitCode;
 	LPCTSTR lpApplicationName;
-	LPTSTR lpCommandLine;
+
+#ifdef _WIN32
+	TCHAR lpCommandLine[200] = _T("cmd /C set");
```

Comment:
Sure this will copy the string to the buffer and not just set a pointer?

Reply:
I think so, at least the unit test passes correctly.